ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -280,13 +280,13 @@ public void setText(String text) {
      * @param animate whether to animate to text.
      */
     public synchronized void setText(String text, boolean animate) {
-        this.text = text;
-        final char[] targetText = text == null ? new char[0] : text.toCharArray();
-
-        if (columnManager.shouldDebounceText(targetText)) {
+        if (text.equals(this.text)) {
```

Comment:
we need to guard against nulls here. might be a good opportunity to add nullability annotations, but might be out of scope of this diff